cmake/strict/strict_flags_helpers.cmake

```diff
@@ -169,6 +169,7 @@ function(get_clang_base_flags flags)
         list(APPEND clang_flags "-Wno-unused-command-line-argument")
     endif()
 
+    # FIXME-WT-8052: Figure out whether we want to disable these or change the code.
```

Comment:
@etienneptl will a file in this directory be caught by your FIXME scripts?

Reply:
Yes, it should! I did run a quick test:
```
> ./s_mentions 
There are comments mentioning wt-8052 in the code, please check if they need to be resolved:
../build_posix/aclocal/strict.m4:158:	# FIXME-WT-8052: Figure out whether we want to disable these or change the code.
../configure:17516:	# FIXME-WT-8052: Figure out whether we want to disable these or change the code.
```